### PR TITLE
I developed Bayesian Nonparametric Chinese Restaurant Process Clustering Tempate.

### PR DESCRIPTION
**Pull request by jirotubuyaki**
I developed Bayesian Nonparametric Chinese Restaurant Process Clustering Template.

https://github.com/jirotubuyaki/predictionio-template-crp-clustering

I update templates.yaml.
I would like to contribute to Apache PredictionIO community and please accept my pull request.

### CODE REVIEW

**Comment by haginot** [comment]
@jirotubuyaki Thank you for adding a new template.
I would like to update PIO documentation after this PR be merged.
For that, could you change a target branch from `develop` to `livedoc`.

Guidelines are here: http://predictionio.incubator.apache.org/community/contribute-documentation/

In short;
* Use the livedoc branch if you want to update the current documentation.
* Use the develop branch if you want to write documentation for the next release.

File: docs/manual/source/gallery/templates.yaml [hunk 1]
```diff
@@ -86,6 +86,20 @@
...
+    repo: "https://github.com/jirotubuyaki/predictionio-template-crp-clustering"
+    description: |-
+      Chinese restaurant process is stochastic process for statistical inference. The clustering which uses Chinese restaurant process does not need to decide the number of clusters in advance. This algorithm automatically adjusts it.
+    tags: [clustering]
+    type: Parallel
+    language: Scala
+    license: "Apache Licence 2.0"
+    status: alpha
+    pio_min_version: 0.10.0-incubating
+    apache_pio_convesion_required: "requires conversion"
```
Comment (dszeto) [inline docs/manual/source/gallery/templates.yaml]: @jirotubuyaki It looks like your template is already resolving PIO artifacts in the Apache package namespace, so it should not require conversion.

Comment (jirotubuyaki) [reply docs/manual/source/gallery/templates.yaml]: Thank you for your comments.
Now I prepare new pull request.

**Comment by jirotubuyaki** [comment]
I changed "apache_pio_convesion_required" parameter and pull request branch.
Thank you for your support.

**Pull request closed without merging**